code/modules/unit_tests/inhands.dm

```diff
@@ -68,8 +68,8 @@
 			if(!lefthand_file)
 				TEST_FAIL("Missing left inhand icon file for [item_path].\n\tinhand_icon_state = \"[held_icon_state]\"[match_message]")
 			else
-				missing_left = !("[lefthand_file]" in possible_icon_states[held_icon_state])
-				if(missing_left && ("[lefthand_file]" in possible_icon_states[""]))
+				missing_left = !icon_exists(lefthand_file, held_icon_state)
+				if(missing_left && icon_exists(lefthand_file, ""))
```

Comment:
I'm a bit confused. Does this mean that if we have our CS:S fallback textures, the test will treat it as okay?

Reply:
Still fails due to the inhand_icon_state var being an invalid value, this just adjusts the output messages to specify if it's using the CS:S fallback & changes which failure log is used to make it easier to read the output.

Reply:
Aaaah ok